### PR TITLE
Fixes issue #2396 and #3537

### LINKED ISSUE #2396: [RTL] The "+" from the intall button has display issues
Steps to reproduce:
1. Load AMO-dev page
2. Go to Other Languages drop-down and pick a RTL language i.e. https://addons-dev.allizom.org/he/firefox/
3. Go to any add-on details page https://addons-dev.allizom.org/he/firefox/addon/facebook-phishing-protector/?src=hp-dl-featured
4. Observe the "+" from the install button 

Expected results:
There are no display or layout issues.

Actual results:
The "+" from the install button has display issues.

Notes/Issues:
Verified on FF45(Win 7). Issue is reproducing on AMO-dev, stage and prod.
Screenshot for this issue:

### LINKED ISSUE #3537: [RTL] layout is broken in theme detail page
### Describe the problem and steps to reproduce it:
1. Go to any theme detail page (https://addons-dev.allizom.org/en-US/firefox/addon/mikink/)

### Actual Result
Buttons are messed up and layout is not in correct way.
before
actual

### Expected Result
There should not be any layout problem issue in theme detail page.
after
expected

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes #3537
Fixes #2396 

# Issue #3537 
## Before
before
## After
after

# Issue #2396 
## Before
Before
## After
after

### CODE REVIEW

**Comment by tofumatt** [comment]
Could you add some before/after screenshots please? Thanks!

File: static/css/zamboni/zamboni.css [hunk 1]
```diff
@@ -2303,6 +2303,9 @@ form .error .note.error {
     width: 320px;
     float: right;
 }
+.html-rtl #persona-summary table{
```
Comment (tofumatt) [inline static/css/zamboni/zamboni.css]: Please add a space before the `{`.

Comment (atiqueahmedziad) [inline static/css/zamboni/zamboni.css]: done :)

**Comment by muffinresearch** [comment]
For future reference to help with review etc:

* Put a descriptive title for the PR (e.g. Update RTL issues in theme detail page).
* Put the issues fixed in the description. (this way it gets automatically linked, whereas it doesn't in the title.)

**Pull request merged**